plugins/dev9ghzdrk/DEV9.cpp

```diff
@@ -65,18 +93,26 @@ static char *libraryName     = "GiGaHeRz's DEV9 Driver"
 #endif
 ;
 
+#ifdef _WIN32
 HANDLE hEeprom;
 HANDLE mapping;
+#else
+int hEeprom;
+int mapping;
+#endif
 
-u32 CALLBACK PS2EgetLibType() {
+EXPORT_C_(u32)
+PS2EgetLibType() {
 	return PS2E_LT_DEV9;
 }
 
-const char* CALLBACK PS2EgetLibName() {
+EXPORT_C_(const)
+char* PS2EgetLibName() {
```

Comment:
Compile error. Should be `EXPORT_C_(const char*)`, not `EXPORT_C_(const) char*`.

Reply:
Corrected